### PR TITLE
Final Routefix and Input Fix

### PR DESCRIPTION
**Pull request by StormCrozier**
## Brief Description
Can now add events and event showings. Updates to UI for adding tickets to a showing, deleting a showing box, and getting a ticket from the main page

Please include a brief description of changes. Please also include relavent information gathered while making these changes, as well as the context behind the changes. Finally, list any dependencies that were changed or added.

## References Issue

Issue #

## Type of change

- [ ] Bug fix
- [X] Feature
- [ ] Major change (potentially breaking, list anything that is broken below)
- [ ] Documentation

### Breaking Changes

Briefly list things that were broken as a result of this PR.

## Testing done

Describe tests that were used, along with instructions for testing.

- [ ] Test1
- [ ] Test2
...

- [X] Docker was used.
- [ ] Docker was not used and I have listed my configuration below.

**System Specifications**: 
* Operating System: macOS Ventura
* Node Version: v18.5.0
* NPM Version: 8.12.1

## Checklist:

- [X] My code follows the styling guidelines.
- [ ] I have added unit tests and verified that they pass.
- [X] I have used the linter and fixed any linting issues.
- [X] I have commented the code.
- [ ] I have adjusted the documentation to match the changed code.
- [X] Prior to submitting the PR, I made sure the latest changes were merged with my branch.
- [X] Existing tests pass locally with changes.

### Explanation for unchecked

Provide a brief explanation for why the checklist was not completed.

## Additional information: 

Provide any additional information that might be useful for this commit.

### CODE REVIEW

File: server/src/api/events/event.service.ts [hunk 1]
```diff
@@ -243,16 +326,16 @@ export const createEvent = async (params: any): Promise<response> => {
             ($1, $2, $3, true, $4, $5)
           RETURNING *;`,
     values: [
-      params.seasonid_fk,
+      params.seasonIdFk,
```
Comment (ben16williams) [inline server/src/api/events/event.service.ts:329]: Can we leave this as `seasonid_fk` to keep it consistent with the db table?

File: server/src/api/events/event.router.ts [hunk 2]
```diff
@@ -290,6 +290,7 @@ eventRouter.post('/instances', checkJwt, checkScopes, async (
   // going to need to use auth0 authentication middleware
   // deleted isAuthenticated function
   try {
+    console.log(req.body);
```
Comment (ben16williams) [inline server/src/api/events/event.router.ts:293]: Could probably remove this console log

Comment (StormCrozier) [reply server/src/api/events/event.router.ts:293]: Removed

File: server/src/api/events/event.service.ts (hunk 1 above)
Comment (StormCrozier) [reply server/src/api/events/event.service.ts:329]: I changed it due to ESlint complaining about camel case notation.  I can change it back but we will get linting errors.

**Review by ben16williams — APPROVED** [review]

**Pull request merged**